src/modules/analysis/AnalysisAddComponent.jsx

```diff
@@ -55,29 +88,42 @@ function newLine() {
   </Form>;
 }
 
+const getVal = (form, field) => form[field] && (form[field].rawValue || ''); // TODO may be inline
```

Comment:
Should we rename this to `formRawValue` or something more informative then `getVal`?